notebooks/grism_efficency_vs_fringe_frequency.ipynb

```diff
@@ -0,0 +1,1399 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Grism Efficiency vs Fringe Frequency\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# external\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import plot\n",
+    "from astropy.units import unit\n",
+    "\n",
+    "# project\n",
+    "from payload_designer.components import diffractors"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "Run this section onwards when a parameter is updated."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# region constants\n",
+    "\n",
+    "a = 90\n",
+    "d = 2.5\n",
+    "eff_mat = 0.85\n",
+    "m = 1\n",
+    "n_1 = 1.0\n",
+    "n_2 = 1.52\n",
+    "n_3 = 1.3\n",
+    "n_g = 0.1\n",
+    "# endregion\n",
+    "\n",
+    "# region variables\n",
+    "a_in = np.linspace(start=0, stop=5, num=5)\n",
+    "l = np.linspace(start=1400, stop=1800, num=5)\n",
+    "v = np.linspace(start=900, stop=1200, num=5)\n",
```

Comment:
```suggestion
```
It doesn't look like these variables are being used anywhere. Perhaps we can remove them?